config.toml

```diff
@@ -10,7 +10,7 @@ theme = ["docsy"]
 enableGitInfo = true
 
 [frontmatter]
-lastmod = []
+lastmod = [":git", ":fileModTime", "lastmod", ":default"]
```

Comment:
FYI @kezhenxu94 We are bringing the timestamp back, but according to the correct time flag.

This PR is still hold because we are trying to find out a way for the docs build, it is cloned from other repos, so, all above seems not working.

The reason we are trying this is for SEO, this header is notifying the search engine, this page changed.